data/json/monsters/zed_children.json

```diff
@@ -261,11 +267,45 @@
                 [ "child_items", 65 ]
             ]
         },
+        "burn_into" : "mon_zombie_child_scorched",
         "hp":25,
         "death_function":["BOOMER"],
         "special_attacks":[["GRAB", 5]],
         "description":"This swollen, gooey looking mutant child looks bad, even for a zombie.  Maybe that's why it seems to want a hug so badly.",
         "//": "GUILT because it still looks enough like a child to evoke pity",
         "flags":["SEES", "HEARS", "STUMBLES", "WARM", "BLEED", "GUILT", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_child_scorched",
+    "name": "burned child zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"z",
+    "color":"brown",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":15,
+    "aggression":100,
+    "morale":100,
+    "speed":100,
+    "melee_skill":2,
+    "melee_dice":2,
+    "melee_dice_sides":1,
+    "melee_cut":2,
+    "dodge":2,
```

Comment:
Dodge is pretty important for zed children, since they get an (undocumented?) +3 dodge from size.

They could get some extra cut damage, though - 3-5 would be fine.